docs/docsite/rst/reference_appendices/YAMLSyntax.rst

```diff
@@ -80,6 +80,8 @@ Ansible doesn't really use these too much, but you can also specify a boolean va
     likes_emacs: TRUE
     uses_cvs: false
 
+Use lowercase 'true' or 'false' for boolean values in dictionaries to ensure good code quality, testable by using yamllint.
```

Comment:
totally disagree with that statement, keeping a consistent boolean value is good, the choosing lower case specifically is not really inherently better then most other choices

Reply:
"Ansible-lint, Yamllint and Flake8 kill opinion wars". (Luke Murphy, author of molecule).

Reply:
Just try yamllinting with the default options.

Reply:
a) does not kill opinion wars, just transfers them to the linter options
b) i already know what yamllint does and it is one of the options we actually override

Reply:
Is that documented?

Reply:
We skip some rules because we don't like them.

Reply:
most linters allow you to skip rules because in many case it is just a convention decided by the linting tool author and does not have solid reasons to go one way or another. 

As a general rule we try not to impose opinions to playbook authors, which is why i have a problem with this sentence.

Reply:
More matter of fact now:
> Use lowercase 'true' or 'false' for boolean values in dictionaries to ensure a code quality testable with default yamllint options.